2023-03_sprint/sprint_parties/contributors.md

```diff
@@ -0,0 +1,302 @@
+read the page {ref}`pr_tutorial` and follow the steps mentioned below:
+
+1. Fork and clone the repo. You can get to the repo on GitHub with the icon on the top right of the page
+1. Create a feature branch.
+1. Copy the contributor card template:
+
+    ```
+    :::{grid-item}
+
+    <img alt="Image for contributor template" src="https://raw.githubusercontent.com/data-umbrella/data-umbrella.github.io/main/images/logo/umb-trans.png" class="rounded-circle">
+
+    Contributor Name
+
+    Contributor Location
+
+    [{fas}`globe`](https://www.dataumbrella.org/)
+    [{fab}`github`](https://github.com/data-umbrella)
+    [{fab}`twitter`](https://twitter.com/DataUmbrella)
+    [{fab}`linkedin-in`](https://www.linkedin.com/company/dataumbrella/)
+    :::
+    ```
+
+1. Open this file: `2023-03_sprint/sprint_parties/contributors.md`
+1. Paste the template to the relevant section and update it with your personal info.
+
+    :::{important}
+    * Copy everything. The opening and closing _3 colon fences_ (`:::`) are necessary
+    * Paste your card **under** the last card available (that is, after the last `:::`)
+      but **above** the end of the grid (that is, before the one with 4 colons `::::`)
+    * Empty lines are important for rendering. The empty line between "Contributor Name"
+      and "Contributor Location" places them in different lines when rendering.
+      The _lack_ of empty line between the multiple icons places them in the same
+      line when rendering the website.
+    * The image can be added to the repo ([`_static/contributors_2023_03` folder](https://github.com/pymc-devs/pymc-data-umbrella/tree/main/_static/contributors_2023_03))
+      as part of the PR or be an external link like in the template.
+      But it needs to be a _square image_.
+    :::
+
+    If you do not want to share a website, GitHub, Twitter and/or LinkedIn,
+    those lines can be deleted, keeping an eye on not leaving empty lines between icons+links.
+
+1. Commit your changes and open a Pull Request
+
+::::
+
+## Returning Contributors
+
+::::{grid} 2 3 4 5
+:gutter: 1
+
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors_2022_07/Lucy_Jimenez.jpeg" class="rounded-circle">
+
+Lucy Jimenez
+
+[{fab}`github`](https://github.com/LucyJimenez)
+[{fab}`linkedin-in`](https://www.linkedin.com/in/lucy-j/)
+[{fab}`twitter`](https://twitter.com/JimenezLucyJ)
+
+:::
+:::{grid-item}
+
+<!-- <img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+
+Alihan Zihna
+
+Dublin (Ireland)
+:::
+
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+
+Juan Pablo Ibieta Jimenez
+
+São Paulo, SP (Brasil)
+
+[{fab}`github`](https://www.github.com/pibieta)
+[{fab}`linkedin-in`](https://www.linkedin.com/in/pibieta)
+[{fab}`twitter`](https://twitter.com/pabloibieta)
+:::
+::::
+
+## New Contributors
```

Comment:
do we want to comment this section out? also, should we add a translators section?